### ISSUE TITLE
Chrome: net::ERR_INSUFFICIENT_RESOURCES on large number of files

### ISSUE THREAD

**Issue by debuglevel** [debuglevel/LiberatePDF2#1]
If you submit a large number of files, Chrome will throw net::ERR_INSUFFICIENT_RESOURCES errors on checking files.

Possible solutions:
- do nothing (if the number I tried out with was just insane)
- throttle checking files (right now, every 1 second all pending files are checked).
  - the interval could be larger, the more files there are.
  - some mechanism to check e.g. 1 file per second on average.
  - now, probably all files are checked in parallel. Could be serial.
- don't check each file, but do a batch check (POST all IDs, get status of them). (how would this look in a RESTful way?)

**Comment by debuglevel** [comment]
On my last check with an unrealistic high number of files, after some time, no more errors have shown and most entries were "done" in the end.
-> rather low priority therefore.
